v1/counterpartymetadata/counterpartymetadata.gunk

```diff
@@ -0,0 +1,2186 @@
+	// PublicAlias is the public_alias metadata made on a other_account.
+	PublicAlias PublicAlias `pb:"1" json:"alias"`
+
+	// BankID is the bank identifier.
+	BankID string `pb:"2" json:"bank_id"`
+
+	// AccountID is the account identifier.
+	AccountID string `pb:"3" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"4" json:"other_account_id"`
+}
+
+type DeletePublicAliasRequest struct {
+	// ID is the public_alias id that will be deleted.
+	ID string `pb:"1" json:"alias"`
+
+	// BankID is the bank identifier.
+	BankID string `pb:"2" json:"bank_id"`
+
+	// AccountID is the account identifier.
+	AccountID string `pb:"3" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"4" json:"other_account_id"`
+}
+
+type GetPublicAliasRequest struct {
+	// ID is the public_alias unique identifier.
+	ID string `pb:"1" json:"alias"`
+
+	// BankID is the bank identifier.
+	BankID string `pb:"2" json:"bank_id"`
+
+	// AccountID is the account identifier.
+	AccountID string `pb:"3" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"4" json:"other_account_id"`
+}
+
+type GetPublicAliasesRequest struct {
+	// BankID is the bank identifier.
+	BankID string `pb:"1" json:"bank_id"`
+
+	// AccountID is the account identifier.
+	AccountID string `pb:"2" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"3" json:"other_account_id"`
+}
+
+type GetPublicAliasesResponse struct {
+	// PublicAliases is the list of public_aliases.
+	PublicAliases []PublicAlias `pb:"1" json:"aliases"`
+
+	// BankID is the bank identifier.
+	BankID string `pb:"2" json:"bank_id"`
+
+	// AccountID is the account identifier.
+	AccountID string `pb:"3" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"4" json:"other_account_id"`
+}
+
+type CreatePrivateAliasRequest struct {
+	// PrivateAlias is the private_alias metadata made on a other_account.
+	PrivateAlias PublicAlias `pb:"1" json:"alias"`
+
+	// BankID is the bank identifier.
+	BankID string `pb:"2" json:"bank_id"`
+
+	// AccountID is the account identifier.
+	AccountID string `pb:"3" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"4" json:"other_account_id"`
+}
+
+type UpdatePrivateAliasRequest struct {
+	// PrivateAlias is the private_alias metadata made on a other_account.
+	PrivateAlias PublicAlias `pb:"1" json:"alias"`
+
+	// BankID is the bank identifier.
+	BankID string `pb:"2" json:"bank_id"`
+
+	// AccountID is the account identifier.
+	AccountID string `pb:"3" json:"account_id"`
+
+	// OtherAccountID is the other_account identifier.
+	OtherAccountID string `pb:"4" json:"other_account_id"`
+}
+
+type DeletePrivateAliasRequest struct {
+	// ID is the private_alias id that will be deleted.
+	ID string `pb:"1" json:"alias"`
```

Comment:
ups, wrong json tag

Reply:
thank you thank you, that one does not need to be there actually.